hadoop-tools/hadoop-azure/dev-support/testrun-scripts/runtests.sh

```diff
@@ -40,8 +41,9 @@ values=("{account name}.dfs.core.windows.net" "true" "SharedKey")
 generateconfigs
 
 combination=NonHNS-SharedKey
-properties=("fs.azure.abfs.account.name" "fs.azure.test.namespace.enabled" "fs.azure.account.auth.type")
-values=("{account name}.dfs.core.windows.net" "false" "SharedKey")
+properties=("fs.azure.abfs.account.name" "fs.azure.test.namespace.enabled" "fs.azure.account.auth.type"
```

Comment:
A new combination called AppendBlob-HNS-OAuth needs to be added. It will have all properties of HNS OAuth + append blob test config. 

Leave current set of combination untouched.

Reply:
Done.